plugins/images/marker-machina.svg

```diff
@@ -0,0 +1 @@
4.31 0c-1.823 1.4922e-5 -3.3012 1.4781-3.3012 3.3012 6.79e-4 0.49756 0.11465 0.98752 0.331 1.4341l-0.0015-0.00155 2.9567 6.1141c1.0296-2.1068 2.0576-4.2143 3.0876-6.3209 0.11283-0.2311 0.22776-0.72488 0.22851-1.2258-1.5e-5 -1.8231-1.4781-3.3012-3.3012-3.3012z" fill="#a00"/><path d="m14.014 0.27855c-1.9309 0.1212-3.3059 2.3428-2.5282 4.1211 0.91397 1.9578 1.8785 3.8929 2.8116 5.842 0.96658-2.038 2.0264-4.0403 2.9363-6.1001 0.57688-1.8146-0.91589-3.8834-2.8274-3.8735-0.13072-0.013742-0.26164 0.0080816-0.39239 0.01052z" fill="#f00"/><path d="m14.049 0.54048c-1.9144 0.095013-3.1853 2.5066-2.1415 4.1307 0.78092 1.6616 1.5927 3.3086 2.3897 4.9626 0.91014-1.9616 2.0047-3.8554 2.7581-5.8817 0.30334-1.7055-1.2591-3.4445-3.0063-3.2116z" fill="url(#linearGradient13037)"/><g transform="matrix(1.0008 0 0 1.0008 -.0089214 -.0087921)"><g transform="matrix(.068108 0 0 .068108 7.1534 -6.8742)" fill="#fff" stroke="#fff"><path d="m105 127.63a20.866 20.866 0 0 0-20.866 20.866 20.866 20.866 0 0 0 20.866 20.866 20.866 20.866 0 0 0 20.867-20.866 20.866 20.866 0 0 0-20.867-20.866zm0 1.4904a19.376 19.376 0 0 1 19.376 19.376 19.376 19.376 0 0 1-19.376 19.376 19.376 19.376 0 0 1-19.376-19.376 19.376 19.376 0 0 1 19.376-19.376z" stroke-width=".26703"/><circle cx="105" cy="148.5" r="16.892" stroke-width=".21617"/><path d="m106.4 149.27-3.3629-24.672 2.2506-0.27661 4.4496-5.5933-0.12715-1e-5 -5.9751-10.424 1.3984 10.043-9.2163 1.3345-0.50864-4.4492-2.7968 5.848 5.5301 4.1313 2.2546-0.27697 3.3705 24.78z" stroke-width=".265"/><g transform="translate(36.276 9.8741)" stroke-width=".265"><path d="m69.885 140.43 15.068-19.824 1.787 1.3958 7.1014-0.80874-0.08992-0.0899 3.1461-11.596-6.1128 8.0905-7.4605-5.5733 2.7864-3.5057-6.1128 2.1575 0.98909 6.8316 1.7901 1.3984-15.139 19.906z"/><path d="m68.984 140.98 24.672-3.3628 0.27662 2.2506 5.5933 4.4496-5e-6 -0.12716 10.424-5.9751-10.043 1.3984-1.3345-9.2163 4.4492-0.50864-5.848-2.7968-4.1313 5.5301 0.27698 2.2546-24.78 3.3705z"/><path d="m67.953 140.74 19.824 15.068-1.3958 1.787 0.80874 7.1014 0.08991-0.0899 11.596 3.1461-8.0905-6.1128 5.5733-7.4605 3.5057 2.7864-2.1575-6.1128-6.8316 0.98908-1.3984 1.7901-19.906-15.139z"/><path d="m67.395 139.84 3.3629 24.672-2.2506 0.27661-4.4496 5.5933 0.12715 1e-5 5.9751 10.424-1.3984-10.043 9.2163-1.3345 0.50864 4.4492 2.7968-5.848-5.5301-4.1313-2.2546 0.27697-3.3705-24.78z"/></g><g stroke-width=".265"><path d="m103.91 148.68-15.068 19.824-1.787-1.3958-7.1014 0.80874 0.08992 0.0899-3.1461 11.596 6.1128-8.0905 7.4605 5.5733-2.7864 3.5057 6.1128-2.1575-0.98909-6.8316-1.7901-1.3984 15.139-19.906z"/><path d="m104.82 148.13-24.672 3.3628-0.27662-2.2506-5.5933-4.4496 5e-6 0.12716-10.424 5.9751 10.043-1.3984 1.3345 9.2163-4.4492 0.50864 5.848 2.7968 4.1313-5.5301-0.27698-2.2546 24.78-3.3705z"/><path d="m105.85 148.37-19.824-15.068 1.3958-1.787-0.80874-7.1014-0.08991 0.0899-11.596-3.1461 8.0905 6.1128-5.5733 7.4605-3.5057-2.7864 2.1575 6.1128 6.8316-0.98908 1.3984-1.7901 19.906 15.139z"/></g></g></g></svg>
```

Comment:
Not sure why, but the SVG doesn't work on mobile and default png is displayed instead 😬

![Screenshot_20240122_044653_IITC Mobile Test.png](https://github.com/IITC-CE/ingress-intel-total-conversion/assets/9021449/06d7b9ce-facf-4a45-bf1e-3f2c0b0b5ece)

Reply:
fixed